test/embedding/embedtest.cc

```diff
@@ -3,6 +3,7 @@
 #endif
 #include "node.h"
 #include "uv.h"
+#include <algorithm>
 #include <assert.h>
```

Comment:
`make lint-cpp` reports the following:
```
test/embedding/embedtest.cc:7:  Found C system header after C++ system header. Should be: embedtest.h, c system, c++ system, other.  [build/include_order] [4]
```
```suggestion
#include <assert.h>

#include <algorithm>
```